### ISSUE TITLE
Quantile function missing for MixtureModels

### ISSUE THREAD

**Issue by tlnagy** [JuliaStats/Distributions.jl#500]
It seems as though there isn't a `quantile` function defined for `MixtureModels`, but `cdf` is defined. It would be nice to have both.

``` .julia
julia> using Distributions

julia> dist = MixtureModel(Normal, [(1, 1), (-1, 1)])
MixtureModel{Distributions.Normal}(K = 2)
components[1] (prior = 0.5000): Distributions.Normal(μ=1.0, σ=1.0)
components[2] (prior = 0.5000): Distributions.Normal(μ=-1.0, σ=1.0)


julia> cdf(dist, 0)
0.5

julia> quantile(dist, 0.5)
ERROR: MethodError: `quantile` has no method matching quantile(::Distributions.MixtureModel{Distributions.Univariate,Distributions.Continuous,Distributions.Normal}, ::Float64)
Closest candidates are:
  quantile(::Distributions.Distribution{Distributions.Univariate,S<:Distributions.ValueSupport}, ::Float64)
  quantile(::Distributions.Distribution{Distributions.Univariate,S<:Distributions.ValueSupport}, ::Real)
  quantile(::AbstractArray{T,1}, ::Any)
  ...
 in quantile at /Users/tamasnagy/.julia/v0.4/Distributions/src/univariates.jl:168

julia> Pkg.status("Distributions")
 - Distributions                 0.8.10

```

**Comment by axsk** [comment]
This is annoying also due to Plots.jl using quantile for plotting distributions, such that
```
plot(MixtureModel([...]))
```
does not work :(

**Comment by devmotion** [comment]
Fixed in Distributions 0.25.16.
